.github/workflows/ubuntu.yml

```diff
@@ -224,9 +224,9 @@ jobs:
           cd misc/
           python_version=${{ matrix.python-version }}
           if [[ $python_version=="3.6" ]]; then
-            tox -e py36,translator,search,style
+            tox -e py36,translator,search,style,cplex,soplex
           elif [[ $python_version=="3.8" ]]; then
-            tox -e py38,translator,search,style
+            tox -e py38,translator,search,style,cplex,soplex
```

Comment:
We don't have to distinguish between the Python version. (The if statement can go away.) Also, instead of py36 and py38, we want to run "driver" and "autodoc" now.

Reply:
Aye. issue1003 which is doing that will be merged first and then I will  rebase.